plugins/controlboard/src/ControlBoardDriver.cpp

```diff
@@ -1270,36 +1270,38 @@ bool GazeboYarpControlBoardDriver::check_joint_within_limits_override_torque(int
 
     int   signKp = (positionPID.GetPGain()>0?1:-1);
     int   signRef = (ref>0?1:-1);
-    //if (m_controlMode[i] == VOCAB_CM_TORQUE || m_interactionMode[i] == VOCAB_IM_COMPLIANT)
-    if (m_controlMode[i] != VOCAB_CM_IDLE && m_controlMode[i] != VOCAB_CM_FORCE_IDLE)
+    if (m_controlMode[i] == VOCAB_CM_TORQUE || m_interactionMode[i] == VOCAB_IM_COMPLIANT)
```

Comment:
To be consistent with the code before this PR, the check should be active also for `VOCAB_CM_PWM` and `VOCAB_CM_CURRENT`, thanks!

Reply:
Do you mean :
if ((m_controlMode[i] == VOCAB_CM_TORQUE || m_interactionMode[i] == VOCAB_IM_COMPLIANT)  ||  (m_controlMode[i] == VOCAB_CM_PWM || m_interactionMode[i] == VOCAB_CM_CURRENT)

Reply:
Yes, given that the logical or is an operation for which the associative property hold, we can just put only a couple of parenthesis.